droid-binary/conf/log4j2.properties

```diff
@@ -1,81 +1,81 @@
-#
-# Copyright (c) 2016, The National Archives <pronom@nationalarchives.gsi.gov.uk>

```

Comment:
The email id should be corrected. It is not strictly part of this PR, so it should be addressed on its own with corrections in all places where it appears in the codebase, also look to see if it forces a copyright year update